corehq/apps/domain/migrations/0016_alter_smsaccountconfirmationsettings_project_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-07-06 12:11
```

Comment:
Can you squash all of the migrations in this project into one?

Reply:
I wasn't aware of this concept. Thanks.

Reply:
perfect, thank you